lib/navigation/navigation.dart

```diff
@@ -1 +1,10 @@
+import 'package:flutter/material.dart';
 
+class Navigation extends StatelessWidget {
+  const Navigation({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(child: Text('Navigation'),);
```

Comment:
add const

Reply:
const can not write, cause i will get from api